servers/audio/effects/audio_effect_record.cpp

```diff
@@ -0,0 +1,101 @@
+/*************************************************************************/
+/*  audio_effect_record.cpp                                             */
```

Comment:
Please keep the `*/` in line on both files.

Thanks for contributing! Very nice feature. :)

Reply:
Fixed. Thanks for spotting it quickly